test/test_prototype_datasets_utils.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import torch
+from datasets_utils import make_fake_flo_file
+from torchvision.datasets._optical_flow import _read_flo as read_flo_ref
+from torchvision.prototype.datasets.utils._internal import read_flo
+
+
+@pytest.mark.parametrize("mode", ("rb", "r+b"))
+def test_read_flo(tmpdir, mode):
+    path = tmpdir / "test.flo"
+    height, width = torch.randint(3, 10, (2,))
```

Comment:
Thanks for the test.
Could we just hardcode h and w?
We should try to avoid RNG without setting the seed in general, but here it seems a bit unnecessary at first sight.